### DISCUSSION TITLE
Issue with ganache-cli as it is depreciated and now ganache

### DISCUSSION THREAD

**Discussion by emilrueh** [smartcontractkit/full-blockchain-solidity-course-py#1779] [General]
In the tutorial video around the timestamp 04:20:00 installing the Ganache CLI is not longer working for me and I found [this thread](https://github.com/trufflesuite/ganache/issues/2440) explaining how ganache-cli is depreciated and should now be: `npm install -g ganache`

**Comment by cromewar** [comment]
Hello @emilrueh 
Thanks for sharing this, this info was also updated for the guide for windows users on the course index.
Thanks again.

**Reply by emilrueh** [reply]
Perhaps it could also be added to the [issues by timecode doc found here](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md) unless that is what was meant by course index. Thanks!

**Reply by cromewar** [reply]
Can you make a PR for this? I'll personally check it.

**Reply by emilrueh** [reply]
When attempting to open a pull request it does not let me due to no different branch and comparison able to be selected. I have opened an issue under the timestamp thread [here](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/issues/1780) and tagged you. Hope this does not double so if it does please delete. (new user here so all not 100% clear, hope to be helpful though!)

**Reply by cromewar** [reply]
I'll add that change personally to the repo, thanks @emilrueh

**Reply by emilrueh** [reply]
Awesome, thank you so much for the help!
